dicoogle/src/main/java/pt/ua/dicoogle/plugins/PluginController.java

```diff
@@ -522,7 +522,8 @@ public StorageInterface getStorageByName(String name, boolean onlyEnabled) {
 
 
     public JointQueryTask queryAll(JointQueryTask holder, final String query, final Object... parameters) {
-        return queryAll(holder, query, parameters);
```

Comment:
Oh boy! I probably never stumbled upon this bug because I usually prefer fetching the `QueryInterface` objects and searching on them directly.

Reply:
Same thought! Well, guess its fixed now!